src/lib.rs

```diff
@@ -805,7 +805,7 @@ pub trait NewRng: SeedableRng {
     /// 
     /// fn foo() -> Result<(), Error> {
     ///     // This uses StdRng, but is valid for any R: SeedableRng
-    ///     let mut rng = StdRng::from_rng(&mut EntropyRng::new())?;
+    ///     let mut rng = StdRng::from_rng(EntropyRng::new())?;
     ///     
```

Comment:
I think the examples and documentation should do `&mut` much more frequently than not, as inexperienced rust developers may find the error message from a missing `&mut` unintuitive.

Reply:
Using a different convention for `from_rng` than the other RNG consumers bugs me a bit, but seems the best option. It sounds like the RFC I opened about reborrowing is being postponed, so we're not going to get a nice uniform solution any time soon; I guess then this is the best we can do.

Reply:
I haven't checked the error message myself, but really that sounds like the only problematic part.  If it's really confusion then we could suggest a rewording as a rustc bug, which get addressed quickly.